pkgs/development/compilers/llvm/6/clang/default.nix

```diff
@@ -19,7 +19,7 @@ let
     nativeBuildInputs = [ cmake python ]
       ++ stdenv.lib.optional enableManpages python.pkgs.sphinx;
 
-    buildInputs = [ libxml2 llvm ]
+    buildInputs = [ libxml2 llvm compiler-rt ]
```

Comment:
C below how I don't want clang to hard-reference know about `compiler-rt`, but if we have to do this `compller-rt` should be a `depsTargetTarget`, because programs emitted by clang, not clang itself, use `compiler-rt`.